mytoyota/models/vehicle.py

```diff
@@ -365,3 +347,118 @@ def _dump_all(self) -> Dict[str, Any]:
             dump[name] = json.loads(data.model_dump_json())
 
         return censor_all(copy.deepcopy(dump))
+
+    def _daily_summaries(self, summary) -> List[Summary]:
```

Comment:
I think all the summary functions should be given a more meaningful function name. At the moment, I can't tell directly from the function name what is being done here.
So instead of `_daily_summaries` it is better to use `_generate_daily_summaries` or similar.